tests/unified/runner.py

```
@@ -0,0 +1,528 @@
+GREEN = "\033[92m"
+RESET = "\033[0m"
+
+JUDGE_MODEL: str = "claude-haiku-4-5"
+
+
+class TestExecutionError(Exception):
+    pass
+
+
+class UnifiedTestExecutor:
+    def __init__(
```

Ruff (0.14.7)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)